### ISSUE TITLE
Hard-wired paths inside fortran programs

### ISSUE THREAD

**Issue by smahanam** [GEOS-ESM/GEOSgcm_GridComp#46]
mk_CatchCNRestarts.F90, mk_LDASsaRestarts.F90 files in GEOScatch_GridComp/mk_restarts/ have files hard-wired from /discover/nobackup

This is going to cause problems outside nccs machines.

**Comment by weiyuan-jiang** [comment]
I thought Justin was dealing with this issue. But I am not sure the status now

**Comment by JustinPerket** [comment]
> I thought Justin was dealing with this issue. But I am not sure the status now

Yup, I was. It's been left alone for a few months in favor of higher-priority items, but is a worthy task

**Comment by smahanam** [comment]
I added Joe.  
A while ago, I removed those paths and pass the path through regrid.pl (so the path was hard-wired in regrid.pl). Joe said that is undesirable obviously. 
I think there is no rush, as long as no one runs the CATCHCN model and GEOSldas outside nccs machines.
